src/admin-views/aggregator/origins/meetup.php

```diff
@@ -134,13 +136,13 @@ class="tribe-bumpdown-trigger tribe-bumpdown-permanent tribe-bumpdown-nohover tr
 
 <?php
 $field              = (object) array();
-$field->label       = __( 'URL:', 'the-events-calendar' );
+$field->label       = __( 'URL', 'the-events-calendar' );
```

Comment:
in some places you are taking out the colon and in others you've left it. Is there an intentional strategy here?